babel/src/test/resources/sql/big-query.iq

```diff
@@ -1051,29 +1051,98 @@ select unix_date(timestamp '2008-12-25') as d;
 
 !ok
 
-# DATE
-# 'date(x) is shorthand for 'cast(x as date)'
-select date('1970-01-01') as d;
+# DATE(year, month, day)
+select date(2022, 11, 15) as d;
 +------------+
 | d          |
 +------------+
-| 1970-01-01 |
+| 2022-11-15 |
 +------------+
 (1 row)
 
 !ok
 
-!if (false) {
-select date(cast(null as varchar(10))) as d;
-+---+
-| D |
-+---+
-|   |
-+---+
+# TIMESTAMP(string)
```

Comment:
big-query.iq was created by pasting in doc from google's site. please continue the tradition.

Reply:
Cool, I think the most recent commit has what you asked for. The BigQuery docs aren't completely self-consistent style-wise but I copied verbatim. Also, the 3 latter tests involve a combination of different functions, so just kept the original descriptive comments there.